data/json/items/gunmod/brass_catcher.json

```diff
@@ -13,6 +13,7 @@
     "color": "light_gray",
     "location": "brass catcher",
     "mod_targets": [ "shotgun", "smg", "rifle", "pistol", "launcher" ],
+    "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "249 ml", "max_contains_weight": "1 kg" } ],
```

Comment:
According to the item descript and crafting recipe, brass catcher is a bag made from rags, so it's probably not rigid.

Reply:
Yeah, I really wasn't sure about that one. 250ml seems a bit much for just a bag, since then it would have a few liters of volume.